src/config/dynamic-theme-fixes.config

```diff
@@ -12150,6 +12150,22 @@ pre {
 
 ================================
 
+patreon.com
+
+INVERT
+a[href^="https://docs.google.com"] img
+a[href^="https://forms.gle"] img
+
+CSS
+div[data-tag="media-container"] > img {
+    filter: brightness(75%);
+}
+
```

Comment:
Does this target all content(e.g. user's uploaded post)?

Reply:
It does, and that's intentional. A lot of YouTube creators will uploaded incredibly bright thumbnails for their videos (since bright == eye catching).

Reply:
Yeah, I cannot allow such fix, we have user settings to deal with that. This is the same of lowering the brightness of youtube videos.

Reply:
Okay, I'll remove these two lines.